qml/DynamicDisplayStatisticsDialog.qml

```diff
@@ -113,17 +137,20 @@ Dialog {
 
                 onActivated: {
                     activeOk = true
-                    controller.update_available_entity_ids(currentText, "getDataDialogSourceEntityId")
-                    sourceEntityId.recalculateWidth()
-                    regenerateSeriesLabel()
+                    updateSources()
                 }
             }
             AdaptiveComboBox {
                 id: sourceEntityId
                 textRole: "nameId"
                 valueRole: "id"
+                displayText: currentIndex === -1
+                             ? ("Please choose a " + getDataDialogSourceEntityId.currentText + "...")
```

Comment:
In general for all `Please choose`: Is it possible to set this in grey or in different format?